frontend/libs/studio-components/src/hooks/useTableSorting.test.tsx

```diff
@@ -0,0 +1,64 @@
+import { useTableSorting } from './useTableSorting';
```

Comment:
Should we add one more test that is testing that `sortedRows` is undefined if enabled is false? 😊

Reply:
Nice catch! Added 😊